### ISSUE TITLE
`System.ArgumentException` during workflow set-up when using v1.3.2

### ISSUE THREAD

**Issue by NicolasT** [dependabot/fetch-metadata#235]
Updating this action from v1.3.1 to v1.3.2 in a job causes the workflow to fail during the `Set up job` phase:

```
Download action repository 'dependabot/fetch-metadata@v1.3.2' (SHA:90ed90dba204fdf8970c1f891b4349c96353f220)
Error: dependabot/fetch-metadata/v1.3.2/action.yml:
Error: dependabot/fetch-metadata/v1.3.2/action.yml: (Line: 18, Col: 113, Idx: 682) - (Line: 18, Col: 141, Idx: 710): While parsing a block mapping, did not find expected key.
Error: System.ArgumentException: Unexpected type '' encountered while reading 'action manifest root'. The type 'MappingToken' was expected.
   at GitHub.DistributedTask.ObjectTemplating.Tokens.TemplateTokenExtensions.AssertMapping(TemplateToken value, String objectDescription)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Fail to load dependabot/fetch-metadata/v1.3.2/action.yml
```

Reverting to v1.3.1 resolves this.

Looking at https://github.com/dependabot/fetch-metadata/blob/v1.3.2/action.yml#L18, it appears those backticks around a YAML identifier may be the culprit.

**Comment by NicolasT** [comment]
What's kinda weird is how in the GitHub web-interface at https://github.com/dependabot/fetch-metadata/blob/v1.3.2/action.yml#L18 the quotes around `true` in the string are rendered as regular single quotes (which would cause the YAML to be invalid, since the whole string is quoted using single quotes...). However, when opening the file in my editor, they appear to be backquotes :thinking:

**Comment by NicolasT** [comment]
Looks like this was fixed already in #232 / https://github.com/dependabot/fetch-metadata/commit/15259f7733b22a285b8b2c02fa7b77a3ede2aa10. So, a new release (or retraction of 1.3.2) would be good :smiley:

**Comment by NicolasT** [comment]
(Dupe of https://github.com/dependabot/fetch-metadata/issues/231)

**Comment by mctofu** [comment]
Thanks for reporting! We just released `v1.3.3`. Let us know if that didn't solve this issue for you.
